### PR TITLE
tweaks in README.md file - wording and ToC

### PR DESCRIPTION
**Pull request by ghost**
**this PR does basically aim at:**
- *reviewing wording in README file*;
- *adding Table of contents to REAME file*;

---

**main reason for that does basically lie within:**
- *improving accessibility of README file;*
- *making file easier to be read;*

---

**the main motivation between such a thing does basically lie withi:**
- *allowing users to better engage with the repository, and by doing so, increasing their interest in contributing to the community*;

---
This change is [Reviewable](https://reviewable.io/reviews/w3c/web-platform-tests/7649)

### CODE REVIEW

**Comment by w3c-bots** [comment]
# Build PASSED

Started: 2017-10-10 14:39:04
Finished: 2017-10-10 14:43:36





View more information about this build on:

- [WPT PR Status](http://pulls.web-platform-tests.org/build/19288)
- [TravisCI](https://travis-ci.org/w3c/web-platform-tests/builds/286074039)

File: README.md [hunk 1]
```diff
@@ -82,14 +101,14 @@ line syntax is:
 ./wpt run product [tests]
 ```
 
-**On Windows**: You will need to preceed the prior command with
+**On Windows**: You will need to proceed the prior command with
```
Comment (gsnedders) [inline README.md]: That's a bogus correction; the existing text is right.

File: README.md [hunk 2]
```diff
@@ -115,18 +134,18 @@ The `wpt` command provides a frontend to a variety of tools for
...
 commands are:
 
-* `wpt serve` - For starting the wpt http server
+* `wpt serve` - For starting the wpt HTTP server
 * `wpt run` - For running tests in a browser
 * `wpt lint` - For running the lint against all tests
 * `wpt manifest` - For updating or generating a `MANIFEST.json` test manifest
 * `wpt install` - For installing the latest release of a browser or
-  webdriver server on the local machine.
+  web driver server on the local machine.
```
Comment (gsnedders) [inline README.md]: should be WebDriver

File: README.md [hunk 3]
```diff
@@ -162,9 +181,9 @@ submodules, you will need to `git submodule update`, as above. If git
 throws an error like:
 
 ```
-fatal: No url found for submodule path 'resources/webidl2/test/widlproc' in .gitmodules
+fatal: No URL found for submodule path 'resources/webidl2/test/widlproc' in .gitmodules
 Failed to recurse into submodule path 'resources/webidl2'
-fatal: No url found for submodule path 'tools/html5lib' in .gitmodules
+fatal: No URL found for submodule path 'tools/html5lib' in .gitmodules
```
Comment (gsnedders) [inline README.md]: Both this and the above are right (because they're copying git's error messages)

File: README.md [hunk 4]
```diff
@@ -189,11 +208,11 @@ line endings, as it will cause lint errors. For git, please set
 Certificates
 ============
 
-By default pregenerated certificates for the web-platform.test domain
+By default generated certificates for the web-platform.test domain
```
Comment (gsnedders) [inline README.md]: Again existing text is right. (Potentially should be hyphenated, but the "correction" is definitely wrong.)

File: README.md [hunk 5]
```diff
@@ -189,11 +208,11 @@ line endings, as it will cause lint errors. For git, please set
...
 are provided in the repository. If you wish to generate new
 certificates for any reason it's possible to use OpenSSL when starting
 the server, or starting a test run, by providing the
-`--ssl-type=openssl` argument to the `wpt serve` or `wpt run`
+`--SSL-type=openssl` argument to the `wpt serve` or `wpt run`
```
Comment (gsnedders) [inline README.md]: This is wrong, given it's a command line argument….

File: README.md [hunk 6]
```diff
@@ -235,15 +254,15 @@ The master branch is automatically synced to http://w3c-test.org/.
 
 Pull requests are
 [automatically mirrored](http://w3c-test.org/submissions/) except those
-that modify sensitive resources (such as `.py`). The latter require
-someone with merge access to comment with "LGTM" or "w3c-test:mirror" to
+that modify sensitive resources (such as `.py`). The latter requires
+someone with merge access to comment with "LGTM" or "w3c-test: mirror" to
```
Comment (gsnedders) [inline README.md]: That's wrong (it's a magic string).

File: README.md [hunk 7]
```diff
@@ -235,15 +254,15 @@ The master branch is automatically synced to http://w3c-test.org/.
...
 indicate the pull request has been checked.
 
 Finding Things
 ==============
 
-Each top-level directory matches the shortname used by a standard, with
-some exceptions. (Typically the shortname is from the standard's
+Each top-level directory matches the short name used by a standard, with
+some exceptions. (Typically the short name is from the standard's
```
Comment (gsnedders) [inline README.md]: This should be one word, as with the existing text. (It's a W3C term-of-art.)

File: README.md [hunk 8]
```diff
@@ -296,7 +315,7 @@ Thank you!
 Lint tool
 ---------
 
-We have a lint tool for catching common mistakes in test files. You
+We have a lint tool for catching common mistakes in text files. You
```
Comment (gsnedders) [inline README.md]: Existing text is right.

File: README.md [hunk 9]
```diff
@@ -305,7 +324,7 @@ your local web-platform-tests working directory like this:
 ```
 
 The lint tool is also run automatically for every submitted pull
-request, and reviewers will not merge branches with tests that have
+the request, and reviewers will not merge branches with tests that have
```
Comment (gsnedders) [inline README.md]: GitHub has "pull requests", not "pull the requests"…

File: README.md [hunk 10]
```diff
@@ -330,7 +349,7 @@ either:
 * the `tools` subdir at the root of the repository, or
 
 * the `tools` subdir at the root of any top-level directory in the
-  repository which contains the tests the script is meant to be used
+  the repository which contains the tests the script is meant to be used
```
Comment (gsnedders) [inline README.md]: Duplicate "the" added.

File: README.md [hunk 11]
```diff
@@ -339,7 +358,7 @@ the URL for a file within them.
 
 If you want to add a script for use with a particular set of tests but
 there isn't yet any `tools` subdir at the root of a top-level
-directory in the repository containing those tests, you can create a
+the directory in the repository containing those tests, you can create a
```
Comment (gsnedders) [inline README.md]: This doesn't make sense with the "the" added here?

File: README.md [hunk 12]
```diff
@@ -2,16 +2,35 @@ The web-platform-tests Project [![IRC chat](https://goo.gl/6nCIks)](http://irc.w
 ==============================
 
 The web-platform-tests Project is a W3C-coordinated attempt to build a
-cross-browser testsuite for the Web-platform stack. Writing tests in a way
+cross-browser test suite for the Web-platform stack. Writing tests in a way
 that allows them to be run in all browsers gives browser projects
 confidence that they are shipping software that is compatible with other
-implementations, and that later implementations will be compatible with
-their implementations. This in turn gives Web authors/developers
+implementations and those later implementations will be compatible with
```
Comment (gsnedders) [inline README.md]: The existing "that" is right here, I think?

File: README.md [hunk 13]
```diff
@@ -2,16 +2,35 @@ The web-platform-tests Project [![IRC chat](https://goo.gl/6nCIks)](http://irc.w
...
+their implementations. This, in turn, gives Web authors/developers
 confidence that they can actually rely on the Web platform to deliver on
 the promise of working across browsers and devices without needing extra
-layers of abstraction to paper over the gaps left by specification
+layers of abstraction to paper over the gaps left by the specification
```
Comment (gsnedders) [inline README.md]: We're talking about specifications generally here, so there should definitely be no "the" here.

**Review by gsnedders — CHANGES_REQUESTED** [review]

**Comment by tiagomoraismorgado88** [comment]
i guys thanks for getting in touch. i will come back to it and review the feedback as soon as i can

**Comment by tiagomoraismorgado88** [comment]
#reverting all changes except table of contents

**from the perspective of reviewer, the overall impression i got was:**
- *that everything was wrong except ToC*

---

**bearing this in mind:**
- *I reverted all the text changes except table of contents*

**looking forward**
**kind regards**

**Pull request closed without merging**